src/Uno.UI/UI/Xaml/AdaptiveTrigger.cs

```diff
@@ -35,20 +35,7 @@ private void UpdateState()
 
 			var isActive = w >= mw && h >= mh;
```

Comment:
Are there any way to force windows size? Or, we assume the test engine would have an host of certain size (eg: 50x50), and design cases around these?

Reply:
I don't think we do... That would be another good case of manual test.